src/App.tsx

```diff
@@ -8,7 +8,9 @@ import {
 import './App.css';
 import DoctorOrders from './views/DoctorOrders/DoctorOrders';
 import Login from './views/Login/Login';
+import axios from 'axios';
 
+axios.defaults.baseURL = process.env.REACT_APP_PIMS_BACKEND_URL ? process.env.REACT_APP_PIMS_BACKEND_URL : 'http://localhost:5051';
```

Comment:
Should the port for the default localhost be process.env.BACKEND_PORT?

Reply:
we could do it that way but then that's just one more env to manage.. would have to break BACKEND_URL into BACKEND_BASE and BACKEND_PORT. I think the less envs there are the easier it is to manage things, what would be the benefit to doing it that way? I can make the change relatively easily if that's something we want to do

Reply:
I meant just update the part with the localhost because the port 5051 is already defined in an environment variable.

Reply:
yeah I know, I will update this and change before we merge